src/main/java/core/basesyntax/strategy/MyFileWriterImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.strategy;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class MyFileWriterImpl implements MyFileWriter {
+    public void write(String report, String filePath) {
```

Comment:
The write method should be marked as 'override' since it is implementing a method from the MyFileWriter interface.